### PR TITLE
feat(p/ufmt): Support `Println`

### PR DESCRIPTION
**Pull request by notJoon**
## Description

Add `Println` function into `ufmt` package. Current gno does not support os.Stdout, so I utilize the `println` built-in function to access the Go's standard output buffer.

### CODE REVIEW

File: examples/gno.land/p/demo/ufmt/ufmt.gno [hunk 1]
```diff
@@ -3,7 +3,43 @@
...
+		case string:
+			strs = append(strs, v)
+		case int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64:
+			strs = append(strs, Sprintf("%d", v))
+		case bool:
+			if v {
+				strs = append(strs, "true")
+			} else {
+				strs = append(strs, "false")
+			}
```
Comment (zivkovicmilos) [inline examples/gno.land/p/demo/ufmt/ufmt.gno:26]: You can simplify this to:
```go
if v {
    strs = append(strs, "true")
    
    continue
}

strs = append(strs, "false")
```

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Review by leohhhn — APPROVED** [review]

**Review by thehowl — COMMENTED** [review]
Okay for now, if we add Sprintln / Sprint in the future this code should be moved there.

We are working towards having an "output", likely not os.Stdout, maybe `debug.Output`. We'll see.

**Comment by deelawn** [comment]
What is the advantage of using `ufmt.Println` over the builtin `println`?  I think `println` also accepts variadic interface arguments and prints them out delimited by spaces.

**Comment by notJoon** [comment]
> What is the advantage of using `ufmt.Println` over the builtin `println`?

During the last workshop in seoul, everyone expected `ufmt` to have a `Println` function and tried to use it. Since it wasn't available, I've received feedback and added it. However, using `println` alone is actually sufficient.

**Pull request merged**